docs/cloud/ops/manual_backups.md

```diff
@@ -0,0 +1,33 @@
+# Manual Backup
+
+Backups can be created on demand through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
+
+## Console 
+
+To create a backup in the console, navigate to the clusters view and click on the create backup icon. Click `Create one off backup`.
```

Comment:
Then, click on `Create one-off backup`

Reply:
Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.